docs/keto/index.md

```diff
@@ -1,45 +1,56 @@
+older, widely adopted permission systems:
 
-## How Ory Permissions works
+- Flexibility: Older permission systems typically rely on a fixed set of permissions and access control rules that are difficult
+  to customize or modify. Ory Permissions provides a flexible data model you can customize to fit a wide range of use cases.
+  Creating custom permission models is aided by Ory Permission Language, a developer-friendly, TypeScript-based configuration
+  language.
 
-### Relationships
+- Scalability: While traditional permission systems often struggle to handle large-scale applications and services with high
+  request volume, Ory Permissions is designed to be highly scalable and handle a large number of concurrent requests without
+  sacrificing performance or reliability.
 
-Ory Permissions operates on and manages relationships. These relationships represent the source of truth from which all
-permissions are derived. Common examples include:
+- Consistency: In older permissions systems, the data is often eventually consistent, which leaves windows of opportunity for
+  potential unauthorized access to resources before the data is correctly propagated to all parts of the system. By implementing
+  Zanzibar, Ory Permissions puts consistency in the spotlight and ensures that permission checks that are at the critical path of
+  user experience, are always answered correctly.
 
-- `User c is the owner of Document d`
-- `User a is a member of Group b`
+With Ory Permissions, you can:
 
-In most cases relationships represent a fact that is already part of your application. To ensure the Ory Permission Server always
-has the most up-to-date information, you should use it as the source of truth for your application as well. There are multiple
-APIs to query and manage relationships.
+- Use Ory Permission Language to create permission models that fit your exact use case - RBAC, ABAC, and beyond.
+- Unify authorization logic in one service that's the single source of truth for access rights across all of your applications.
+- Be tech-stack agnostic and use Ory Network [SDKs](./sdk/01_overview.md) available for all major programming languages.
+- Issue fine-grained permissions, for example `user x can read document y`.
+- Allow permissions inheritance through groups, roles, and hierarchies to ensure organic scaling that follows the growth of your
+  application.
 
-### Ory Permission Language
+## Ory Permission Language
 
-The Ory Permission Language (OPL) defines the rules by which permissions are derived from relationships. Every application has
-different rules, so this allows you to customize the behavior to your permission model. Examples of rules are:
+The Ory Permission Language (OPL) is a developer-friendly, [TypeScript](https://www.typescriptlang.org/)-based configuration
+language designed to have a low learning curve and a familiar source in contrary to other, proprietary languages used to represent
```

Comment:
```suggestion
language designed to be easy to learn and have a familiar source in contrary to other, proprietary languages used to represent
```

Reply:
Learning curve metaphors are really broken IMHO, as a steep curve is actually a good thing (you learn things quickly), but has a bad connotation.

Reply:
Updated with a version that mixes your and Patrik's suggestion :)